en/04_Changelogs/5.0.0.md

```diff
@@ -26,9 +34,65 @@ title: 5.0.0 (unreleased)
+[silverstripe/ckan-registry](https://packagist.org/packages/silverstripe/ckan-registry) | CMS5 compatible without support | 
+[silverstripe/comment-notifications](https://packagist.org/packages/silverstripe/comment-notifications) | CMS5 compatible without official support | 
+[silverstripe/comments](https://packagist.org/packages/silverstripe/comments) | CMS5 compatible without official support | 
+[silverstripe/content-widget](https://packagist.org/packages/silverstripe/content-widget) | CMS5 compatible without support | 
+[silverstripe/fulltextsearch](https://packagist.org/packages/silverstripe/fulltextsearch) | CMS5 compatible without support | 
+[silverstripe/graphql-devtools](https://packagist.org/packages/silverstripe/graphql-devtools) | CMS5 compatible without official support | 
+[silverstripe/html5](https://packagist.org/packages/silverstripe/html5) | Folded back in core | 
+[silverstripe/postgresql](https://packagist.org/packages/silverstripe/postgresql) | CMS5 compatible without support | 
+[silverstripe/recipe-ccl](https://github.com/silverstripe/recipe-ccl) | CMS5 compatible without support | 
+[silverstripe/recipe-solr-search](https://github.com/silverstripe/recipe-solr-search) | CMS5 compatible without support | 
+[silverstripe/spellcheck](https://packagist.org/packages/silverstripe/spellcheck) | Dropped | 
+[silverstripe/sqlite3](https://packagist.org/packages/silverstripe/sqlite3) | CMS5 compatible without support | 
+[silverstripe/widgets](https://packagist.org/packages/silverstripe/widgets) | CMS5 compatible without official support | 
+[tijsverkoyen/akismet](https://github.com/tijsverkoyen/akismet) | Dropped | 
+[tractorcow/classproxy](https://github.com/tractorcow/classproxy) | CMS5 compatible without official support | 
+[tractorcow/silverstripe-proxy-db](https://packagist.org/packages/tractorcow/silverstripe-proxy-db) | CMS5 compatible without official support |
+[undefinedoffset/sortablegridfield](https://packagist.org/packages/undefinedoffset/sortablegridfield) | Dropped | [symbiote/silverstripe-gridfieldextensions](https://packagist.org/packages/symbiote/silverstripe-gridfieldextensions) provides equivalent functionality.
+
+## Fixed dependencies {#fixed-depdencies}
+
+Silverstripe CMS relies on many third party dependencies to implement various features. Silverstripe CMS defines many *fixed dependencies* that provide certainty to Silverstripe CMS projects and minimise the risk of composer conflicts.
+
+We defined a list of *fixed dependencies* for the lifetime of Silverstripe CMS 5. Those dependencies will not be upgraded to new major version within the Silverstripe CMS 5 release line.
+
+Review Silverstripe CMS 5 fixed dependencies for more details.
+
 ## API changes {#api-changes}
 
-This is a major release and as a result there are a number of breaking API changes. For a full list of these see [upgrading your project](/upgrading/upgrading_your_project). Some specific details about a few of them are below.
```

Comment:
My view is that the full list should be here.

Reply:
I moved some of that language to the top of the changelog.

Reply:
I disagree - I think we should move the full list of changes into a separate document.
If they're all in the main changelog, people have to scroll past a GIANT WALL of text to see the relevant changes. It's going to be a much bigger wall once Steve has finished auto-generating the full list of changes. I think we should move all of the "removed this deprecated class, changed this to private" etc into a separate document that people can look at _if they're interested and find that relevant_.

The large majority of people will either not look at it at all, or will only search for specific things that are relevant to their project. Having it in the main changelog makes people less likely to bother scrolling to see the changes we have chosen to explicitly call out in our hand-written bits.

Reply:
The approach we've generally taken over the years is to put everything in one file. Yes, it's a big wall of text. My instinct would be to ship it like that for the beta and ask devs if they have a preference. We can re-adjust for stable.

Reply:
We've not done this before - not since CMS 4.0.0. I think it's okay for us to change an approach for something like this.

> ask devs if they have a preference

Do we have a clear list of who we're asking and how we're asking them - and when? And when we'll action the response?

Reply:
I don't think we have to make this a massive consultation. Let's maybe round up a dozen devs and see what they think. We'll be doing that anyway in our bespoke/community outreach.

Reply:
https://github.com/silverstripe/developer-docs/issues/151